Stratis.Bitcoin/BlockPulling/BlockPullerBehavior.cs

```diff
@@ -46,12 +46,12 @@ public class BlockPullerBehavior : NodeBehavior, IBlockPullerBehavior
         /// Token that allows cancellation of async tasks.
         /// It is used during component shutdown.
         /// </summary>
-        public CancellationTokenSource CancellationTokenSource => this.cancellationToken;
+        public CancellationTokenSource CancellationTokenSource { get { return this.cancellationToken; } }
```

Comment:
This is odd, we normally allow expression properties

Reply:
MSVS suggest to do it like this. I mean if we are OK with the `=>` on properties, maybe there is something we can do about the MSVS suggestions so they don't appear.

Reply:
I also prefer the `=>` in this case.

Reply:
i don't really have a preference except that i like the compiler not to report anything to me, be it error, warning or message